app/extensions/brave/content/scripts/flashListener.js

```diff
@@ -20,18 +22,18 @@ function isAdobeLink (href) {
       }
     })
   }
-  var observer = new window.MutationObserver(function (mutations) {
+  setTimeout(() => {
+    var observer = new window.MutationObserver(function (mutations) {
+      replaceAdobeLinks()
+    })
     replaceAdobeLinks()
```

Comment:
I prefer perf over better flash placeholder, might be worth filing an issue though to investigate further.